test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -116,5 +118,95 @@ def fn(x):
         self.assertEqual(x.grad, x_clone.grad)
 
 
+@skipIfTorchDynamo("Not a torch._dynamo test")
+class TestInvokeSubgraphCompile(TestCase):
+    def test_simple(self):
+        def gn(x, y):
+            return (torch.mul(x, y),)
+
+        def fn(x, y):
+            return create_invoke_subgraph_op(gn, (x, y))[0]
```

Comment:
So, is the user API? Or is this just an API we're using for testing?

Reply:
If this is the user API, then I want to bikeshed it more, ask for docs, and also ask for input validation (in the previous PR)

Reply:
It also affects the testing strategy. If this is the frontend, then it should also do something reasonable if the user passes an invalid function to it (like a function that has a graph break in it)

Reply:
Deleted this function for now. We can introduce a user-facing one later on.